### PR TITLE
Medium p 1

### PR DESCRIPTION
**Pull request by agavriluta**
Task : 
" Write a simple program that reads two lines: a line of words and a line containing the search word. The program must search in the first line for a word specified in the second one. The program should output the index of the specified word. If there is no such word in the first line, the program should print Not Found."

### CODE REVIEW

File: MediumProject/src/main/java/stage1/Searching.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        Scanner scan = new Scanner(System.in);
+        return scan.nextLine();
+    }
+
+    public static void search(){
+        String[] arr = inputLine();
+        String word = inputWord();
+        String result = " ";
+        for (int i = 0; i < arr.length; i++) {
+            if (arr[i].equals(word)) result = Integer.toString(i+1);
```
Comment (ppanasencoGD) [inline MediumProject/src/main/java/stage1/Searching.java:22]: why? Integer.toString(i+1)

Comment (ppanasencoGD) [reply MediumProject/src/main/java/stage1/Searching.java:22]: okay you should return index

Comment (ppanasencoGD) [reply MediumProject/src/main/java/stage1/Searching.java:22]: just for alternative

        var var1 =IntStream.range(0, arr.length)
                .filter(index -> arr[index].equals(word))
                .mapToObj(index -> Integer.valueOf(index).toString())
                .findFirst()
                .orElse("Not found");
        System.out.println(var1);

Comment (agavriluta) [reply MediumProject/src/main/java/stage1/Searching.java:22]: ok , thanks

**Review by ppanasencoGD — APPROVED** [review]
Please squash commits before merge

**Pull request merged**